src/main/java/tommy/Ui.java

```diff
@@ -0,0 +1,177 @@
+package tommy;
+
+import java.util.Scanner;
+
+import tommy.task.Task;
+import tommy.task.TaskList;
+
+import tommy.exception.TommyException;
+
+/**
+ * Represents the class that deals with all displays and interactions with the user.
+ */
+public class Ui {
+
+    private static final String name = "Tommy";
```

Comment:
Remember to follow the SCREAMING_SNAKE_CASE format for your constants!

```suggestion
    private static final String NAME= "Tommy";
```